src/sql/plot.py

```diff
@@ -392,6 +394,21 @@ def histogram(
     """
     if not conn:
         conn = sql.connection.ConnectionManager.current
+    if isinstance(breaks, list):
+        if len(breaks) < 2:
+            raise ValueError(
+                f"Breaks given : {breaks}. When using breaks, please ensure "
```

Comment:
`%sqlplot histogram --table penguins.csv --column body_mass_g --breaks 9000` : this prints entire stack trace : 
```
File ~/miniforge3/envs/jupysql/lib/python3.10/site-packages/sql/plot.py:399, in histogram(payload, table, column, bins, with_, conn, category, cmap, color, edgecolor, ax, facet, breaks)
    397 if isinstance(breaks, list):
    398     if len(breaks) < 2:
--> 399         raise ValueError(
    400             f"Breaks given : {breaks}. When using breaks, please ensure "
    401             "to specify at least two points."
    402         )
    403     if not all([b2 > b1 for b1, b2 in zip(breaks[:-1], breaks[1:])]):
    404         raise ValueError(
    405             f"Breaks given : {breaks}. When using breaks, please ensure that "
    406             "breaks are strictly increasing."
    407         )

ValueError: Breaks given : [9000.0]. When using breaks, please ensure to specify at least two points.
If you need help solving this issue, send us a message: https://ploomber.io/community
```

Is this expected behaviour?

Reply:
Changed to `exceptions.ValueError`